src/main/java/com/kodilla/ecommercee/mapper/ProductMapper.java

```diff
@@ -13,13 +13,17 @@
     public Product mapToProduct(final ProductDto productDto) {
         return new Product(productDto.getId(),
                 productDto.getName(),
-                productDto.getPrice());
+                productDto.getPrice(),
+                productDto.getQuantity()
+                );
```

Comment:
brzydkie formatowanie niespójne z pozostałymi fragmentami kodu

Reply:
Nie wiem - jakie formatowanie Ci się spodoba, bo nie ma jednoznacznego formatowania w projekcie. Zmieniłem to na takie jakie mi się wydawało. A jeśli nie, to może powiedz, gdzie ten nawias ze średnikiem winien być?

Reply:
chodzi mi o wcięcie przed ); które ustawiłeś na poziomie productDto. Zobacz, że w pozostałych przypadkach formatowanie robisz tak, że ); ma mniejsze wcięcie niż productDto.

Reply:
gdzie indziej jeszcze pisałeś od razu productDto.getQuantity()); 
Nie chodzi o to co mi się podoba czy nie - chodzi o to, żeby było spójne i wszędzie tak samo.